### PR TITLE
Added etiquette to development guide

### LINKED ISSUE #3: Etiquette in development guide has practically nothing
It currently (in my local machine) only has commit etiquette. That is quite lacking. I will add PR etiquette, issue etiquette and probably more

### PR DESCRIPTION
**Pull request by Exerra**
Added an etiquette section to development guide.
Etiquette section currently only has a commit etiquette, but I have plans on adding a Pull Request etiquette and possibly a few more etiquetets.

### CODE REVIEW

**Comment by Exerra** [comment]
I will work on the PR etiquette now

**Comment by Exerra** [comment]
Finished work on the PR etiquette, will work on issue etiquette.

**Comment by Exerra** [comment]
Quickly made dynamic "What to work on" section

**Comment by Exerra** [comment]
Will now make the issue etiquette

**Comment by Exerra** [comment]
Reviewers, review the site here [https://exerra-discord.github.io/karen-bot-docs/](https://exerra-discord.github.io/karen-bot-docs/) and comment your opinions

**Review by levichlev — COMMENTED** [review]
I would make main branch protected, so no one, even the Amelia herself can't make accidental commit to a wrong branch and changed PR instructions accordingly

File: development/etiquette.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
+# Etiquette
+> Karen has an etiquette? Strange 🤔
+
+Listen up everyone, this will address the Karen Bot etiquette
+
+## Commit etiquette
+- Only push small commits (e.g if you changed 2 files, commit one file, commit the other and only <i>then</i> push) 🤏
+- Keep commit titles short and explain them in the description of the commit 
+- **DO NOT PUSH TO MASTER BRANCH!** Push to a new branch (e.g username-whatyouchanged and create a PR)
```
Comment (Luki120) [inline development/etiquette.md]: Don't be racist lady, change that master wording for main, since your branch is also correctly named. Also in the main readme of Karen repository now that I remember.

File: development/etiquette.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+
+## Pull Request etiquette
+So, this is the etiquette for Pull Requests. Pull Requests are a very useful thing for maintainers and it ensures that the correct people can review your code before it goes into production. That said, let's lay out the rules.
```
Comment (Luki120) [inline development/etiquette.md:15]: Rather than "rules" I'd rephrase it to "conditions" or "conditions for contributing", sounds more friendly imo.

File: development/etiquette.md [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+
+A good example is available on [github.com/Exerra-Discord/karen-bot-docs/pull/1](https://github.com/Exerra-Discord/karen-bot-docs/pull/1) 😅
+
+### Metadata
+- Add the correct labels 🏷️
+- If it is related to a milestone, add it to a milestone 🏁
+- If it is related to an open issue, link it to that issue ❌
+- Assign everyone who worked at that pull request 👩🏾‍🤝‍👩🏻
```
Comment (Luki120) [inline development/etiquette.md]: Who worked **on** that PR, rather than at.

File: development/etiquette.md [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+
+### Actual PR contents
+- Write a proper title that briefly explains what it does
+- Write a detailed description of what you did (and what you will do if you selected "Draft Pull Request")
+
+### Draft PR rules
+- When you're done, click "Ready to review"
+
+## Issue etiquette
+Oh look, the etiquette for Issues! Issues are a very essential thing for highlighting issues in a project. Wether it be a bug, a typo in documentation, something not being there and so on, Issues is the way to go. So, let's look at the Karen Bot etiquette for them!
```
Comment (Luki120) [inline development/etiquette.md:44]: Wether **it would/it could**

**Review by Luki120 — APPROVED** [review]
Check my comments and all good (:

File: development/etiquette.md (hunk 1 above)
Comment (Exerra) [reply development/etiquette.md]: Æ fine

File: development/etiquette.md (hunk 2 above)
Comment (Exerra) [reply development/etiquette.md:15]: I like "rules" more. "conditions" is too friendly for Karen Bot

File: development/etiquette.md (hunk 3 above)
Comment (Exerra) [reply development/etiquette.md]: Yeah alright true

File: development/etiquette.md (hunk 4 above)
Comment (Exerra) [reply development/etiquette.md:44]: non

**Comment by Luki120** [comment]
Hm also, the "Get Started" button should prompt me to the setup instructions section, that's what I was expecting from the get started phrase. Would make more sense to make that change or rephrase it.

**Comment by Exerra** [comment]
> Hm also, the "Get Started" button should prompt me to the setup instructions section, that's what I was expecting from the get started phrase. Would make more sense to make that change or rephrase it.

If the other reviewers also think of that, I will change the contents
If not, I won't

File: development/etiquette.md (hunk 4 above)
Comment (Luki120) [reply development/etiquette.md:44]: Woe

**Comment by Luki120** [comment]
Levichlev back me up here plzz haha

**Comment by Exerra** [comment]
Fixed issues brought to light by @Luki120 which I deemed actual issues

File: development/etiquette.md (hunk 1 above)
Comment (cstanze) [reply development/etiquette.md]: Love the current PR etiquette. Does the `whatyouchanged` part include a suggested change quantifier (major change, minor change, patch)?

**Comment by cstanze** [comment]
Proposing adding `major`, `minor`, and `patch` labels for Pull Requests. Also, pushing to use either SemVer or CalVer versioning.

**Comment by cstanze** [comment]
```diff
diff --git a/development/etiquette.md b/development/etiquette.md
index c92c786..d7333df 100644
--- a/development/etiquette.md
+++ b/development/etiquette.md
@@ -1,19 +1,23 @@
 # Etiquette
+
 > Karen has an etiquette? Strange 🤔
 
 Listen up everyone, this will address the Karen Bot etiquette
 
 ## Commit etiquette
-- Only push small commits (e.g if you changed 2 files, commit one file, commit the other and only <i>then</i> push) 🤏
-- Keep commit titles short and explain them in the description of the commit 
-- **DO NOT PUSH TO MAIN BRANCH!** Push to a new branch (e.g username-whatyouchanged and create a PR)
+
+- Only push small commits (e.g if you changed 2 files, commit one file, commit the other and only *then* push) 🤏
+- Keep commit titles short and explain them in the description of the commit
+- **DO NOT PUSH TO MAIN BRANCH!** Push to a new branch (e.g `username-whatyouchanged` and create a PR)
 
 ## Pull Request etiquette
+
 So, this is the etiquette for Pull Requests. Pull Requests are a very useful thing for maintainers and it ensures that the correct people can review your code before it goes into production. That said, let's lay out the rules.
 
 A good example is available on [github.com/Exerra-Discord/karen-bot-docs/pull/1](https://github.com/Exerra-Discord/karen-bot-docs/pull/1) 😅
 
-### Metadata
+### PR Metadata
+
 - Add the correct labels 🏷️
 - If it is related to a milestone, add it to a milestone 🏁
 - If it is related to an open issue, link it to that issue ❌
@@ -27,16 +31,20 @@ Example:
 ![Example of PR metadata](https://cdn.exerra.xyz/files/png/tutorials/development_guide/etiquette/pr/metadata.png)
 
 ### Actual PR contents
+
 - Write a proper title that briefly explains what it does
 - Write a detailed description of what you did (and what you will do if you selected "Draft Pull Request")
 
 ### Draft PR rules
+
 - When you're done, click "Ready to review"
 
 ## Issue etiquette
+
 Oh look, the etiquette for Issues! Issues are a very essential thing for highlighting issues in a project. Wether it be a bug, a typo in documentation, something not being there and so on, Issues is the way to go. So, let's look at the Karen Bot etiquette for them!
 
-### Metadata
+### Issue Metadata
+
 - Add the correct labels 🏷️
 - If it is related to a project, link that project 🧑🏽‍🤝‍🧑🏼
 - If it is related to a milestone, add it to a milestone 🏁
@@ -45,5 +53,6 @@ Oh look, the etiquette for Issues! Issues are a very essential thing for highlig
 - Don't turn off notifications, please ✉️
 
 ### Actual issue contents
+
 - Write a proper title that briefly explains what the issue is meant for
 - Write a long and detailed description of what the issue is, how you think it could be solved and IF you would like to solve it. All of that info greatly helps the maintainers of Karen Bot determine the right way to handle it.

```

**Comment by cstanze** [comment]
uhh vscode is being kinda stupid rn...

**Review by cstanze — APPROVED** [review]
Looks good to me +1

**Comment by Exerra** [comment]
Yey ok time to merge
